.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -14,30 +14,51 @@ env:
   PYTHONPATH: ${{ github.workspace }}/src/promptflow
   IS_IN_CI_PIPELINE: "true"
 jobs:
-  authorize:
-    environment:
-      # forked prs from pull_request_target will be run in external environment, domain prs will be run in internal environment
-      ${{ github.event_name == 'pull_request_target' &&
-      github.event.pull_request.head.repo.full_name != github.repository &&
-      'external' || 'internal' }}
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        pythonVersion: ['3.9']
```

Comment:
why do we need to do this as separate step?

Reply:
Ok, let me remove this matrix, I previously think that we may find some problems in build stage. And now we could find it totally in testing stage.

Reply:
Separate build stage to prevent there are some unnecessary packages included inside testing environment.